influence_benchmark/RL/SFT.py

```diff
@@ -93,8 +93,10 @@ def formatting_prompts_func(example):
     print_trainable_parameters(model)
```

Comment:
a lot of code is duplicated between KTO and SFT maybe we should make a superclass?